controllers/core/node_controller.go

```diff
@@ -113,7 +127,13 @@ func (r *NodeReconciler) SetupWithManager(mgr ctrl.Manager, healthzHandler *rcHe
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&corev1.Node{}).
 		WithOptions(controller.Options{MaxConcurrentReconciles: MaxNodeConcurrentReconciles}).
-		Complete(r)
+		Watches(
```

Comment:
If you're an owner, can you use .Owns()?

Reply:
Owns() works. Simplify the code further. Thanks